go.mod

```diff
@@ -50,6 +50,7 @@ require (
 	github.com/aerospike/aerospike-client-go/v6 v6.4.0 // indirect
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.11.3 // indirect
 	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl v0.62.0 // indirect
+	gopkg.in/yaml.v2 v2.4.0 // indirect
 )
 
 require (
```

Comment:
It's possible this could use a `go mod tidy`? Just seems weird that there's two blocks of indirect requires but this appears to have already been here so may not be fixable with a tidy.

Reply:
`go mod tidy`  didn't change anything, but blowing away `go.mod` and re-generating it does produce some changes, so let's tackle this in a follow up.